docs/app/get-started/install-cypress.mdx

```diff
@@ -116,7 +116,7 @@ Cypress supports running under these operating systems:
 
 Cypress requires [Node.js](https://nodejs.org/) in order to install. We support the versions listed below:
 
-- **Node.js** 18.x, 20.x, 22.x and above
+- **Node.js** 20.x, 22.x, 24.x and above
```

Comment:
The statement below:

Accordingly, use of Node.js 18 and 23 with Cypress is deprecated and support is planned for removal in a future release of Cypress.

needs to be removed.

Reply:
Addressed here: [`8158b20` (#6137)](https://github.com/cypress-io/cypress-documentation/pull/6137/commits/8158b201d648a47bea5d4e409b745e730538e3ef)